deployment/sw360/docker-entrypoint.sh

```diff
@@ -84,6 +84,34 @@ if [ "$HTTPS_HOSTS" ]; then
 fi
 
 ################################################################################
+# Setup for trusted Certificate Authorities
+#
+# $TRUSTED_CACERTS should be a comma separated list of environment variable
+# names, e.g.:
+# "TRUSTED_CA1,TRUSTED_CA2"
+# Mentioned variables ($TRUSTED_CA1 and $TRUSTED_CA2 in this example) should
+# contain the certificates, e.g.:
+# -----BEGIN CERTIFICATE-----
+# AAAAAAAAAAAAAAAAAAAAAAAAAAAA
+# -----END CERTIFICATE-----
+if [ "$TRUSTED_CACERTS" ]; then
+    for CERT_NAME in $(echo $TRUSTED_CACERTS | sed "s/,/ /g"); do
+        if [ "${!CERT_NAME}" ]; then
+          echo "Trust certificate $CERT_NAME ..."
+          while read -r line; do
```

Comment:
I have encountered the same problems.

--

The old line mentioned above does something else and should not be changed